hack/openstack-csp-helper/README.md

```diff
@@ -0,0 +1,5 @@
+This chart can be used to create a new namespace and two secrets for the clusterstacks approach. It reads clouds.yaml files in its raw form either with username and password or with an application credential. The chart is intended to be used once per Openstack-Project/Tenant. It is meant to prepare one corresponding namespace in the cluster-API management cluster (1:1 relation between openstackproject and cluster-namespace). The recommended way to invoke the chart is:
+
+```
+helm upgrade -i <tenant>-credentials -n <tenant> --create-namespace https://github.com/SovereignCloudStack/cluster-stacks/releases/download/openstack-alpha-1-28-v3/csp-helper-chart.tgz -f clouds.yaml
```

Comment:
How will this be part of the assets? I don't like *openstack-alpha-1-28-v3* in there. Should we release this chart e.g. in github packages if it possible? ... I don't know

Reply:
i dont like it either, the quick and dirty approach that comes to my mind is to keep a zipped version of it in the repo and use a raw.githubusercontent... link. It is ugly to host blobs in a git-repo but for this repo this will not be the only place where this is going to happen

Reply:
Or create a separate release for that, but always it will be `.tgz`.

Reply:
yes i can do that too.